src/coreclr/tools/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -3590,9 +3593,13 @@ void MethodContext::dmpGetThreadLocalStaticBlocksInfo(DWORD key, const Agnostic_
 {
     printf("GetThreadLocalStaticBlocksInfo key %u, value tlsIndex-%016" PRIX64
            ", offsetOfThreadLocalStoragePointer-%u, offsetOfMaxThreadStaticBlocks-%u"
-           ", offsetOfThreadStaticBlocks-%u offsetOfGCDataPointer-%u",
+           ", offsetOfThreadStaticBlocks-%u, offsetOfGCDataPointer-%u"
+           ", value tlsGetAddrFtnPtr-%016" PRIX64 ", tlsIndexObject-%016" PRIX64
+           ", threadVarsSection-%016" PRIX64 ,
            key, value.tlsIndex.handle, value.offsetOfThreadLocalStoragePointer,
-           value.offsetOfMaxThreadStaticBlocks, value.offsetOfThreadStaticBlocks, value.offsetOfGCDataPointer);
+           value.offsetOfMaxThreadStaticBlocks, value.offsetOfThreadStaticBlocks,
```

Comment:
Use `SpmiDumpHelper::DumpAgnostic_CORINFO_CONST_LOOKUP` for `value.tlsIndex`; rearrange these to be in the order they are declared in `Agnostic_GetThreadLocalStaticBlocksInfo`